CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 * Added `CameraCaptureSource`, `CaptureSourceError`, `CaptureSourceObserver`, `VideoCaptureFormat`, and `VideoCaptureSource` interfaces and enums to facilitate releasing capturers as part of the SDK.
 * Added `DefaultCameraCaptureSource` implementation of `CameraCaptureSource`.
 * Added `listVideoDevices` and `listSupportedVideoCaptureFormats` to `MediaDevice.Companion`.
+* Added TURN uris received callback.
```

Comment:
I think builder can use more information on this TURN request improvements

Reply:
I dont think this line is needed any more?

Reply:
Since we're adding a callback, I think it still makes sense to mention it here.